### PR TITLE
Automated cherry pick of #15908

### PR DESCRIPTION
**Pull request by j3ffml**
Cherry pick of #15908 on release-1.1.

### CODE REVIEW

**Comment by jlowdermilk** [comment]
@kubernetes/kubectl

File: docs/user-guide/kubectl/kubectl_rolling-update.md [hunk 1]
```diff
@@ -1,5 +1,33 @@
 <!-- BEGIN MUNGE: UNVERSIONED_WARNING -->
 
+<!-- BEGIN STRIP_FOR_RELEASE -->
```
Comment (janetkuo) [inline docs/user-guide/kubectl/kubectl_rolling-update.md]: We should not have this in release branch, right?

Comment (jlowdermilk) [inline docs/user-guide/kubectl/kubectl_rolling-update.md]: Done.

**Comment by janetkuo** [comment]
@k8s-bot unit test this

**Comment by jlowdermilk** [comment]
@k8s-bot e2e test this issue: #20570

**Comment by lavalamp** [comment]
@ixdy It looks like test output is missing? The XML file says that TestPluginTearDownHook from k8s.io/kubernetes/pkg/kubelet/network/exec failed, but no logs got uploaded here: https://pantheon.corp.google.com/storage/browser/kubernetes-jenkins/pr-logs/pull/20180/kubernetes-pull-test-unit-integration/12727/

**Comment by ixdy** [comment]
@lavalamp the [PR to collect stdout from tests](https://github.com/kubernetes/kubernetes/pull/15129) never got cherry-picked into `release-1.1`.

And `TestPluginTearDownHook` is a known-flake on `release-1.1`: https://github.com/kubernetes/kubernetes/issues/14891

**Comment by bgrant0607** [comment]
This doesn't look like a P0 that should be merged into 1.1.

**Comment by jlowdermilk** [comment]
@bgrant0607 it was supposed to have been included in 1.1 from the beginning but somehow missed the cut. But if there is pushback, 1.2 is close enough. Motivation for cherrypick was users getting bit by doing a rolling update with the same image.

**Comment by jlowdermilk** [comment]
Closing since 1.1 is only taking p0 bugfixes.

**Pull request closed without merging**